src/command.rs

```diff
@@ -297,6 +297,7 @@ fn crc(src: &[u8]) -> u16 {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::constants::ATCA_CMD_SIZE_MAX;
```

Comment:
why did you have to import this where it was not previously required?

Reply:
It is already required but currently the unit test is not compiling. Just run `cargo test` from the root on the current main branch and you will get the error. I just fixed the compilation error because my local linter was complaining. But now it is failing because the results don't match. But that is not the result from this PR so I haven't looked into it.

Reply:
hmm.. that would imply some thing quite wrong.. Trying to see what the info command doesn't encode like it used to anymore